mem_test/README.md

```diff
@@ -30,14 +30,14 @@ might lead to unpredictable behavior in concurrent programs.
 
 3. Build bsa-acs UEFI app <br>
 Note :  Install GCC-ARM 13.2 [toolchain](https://developer.arm.com/downloads/-/arm-gnu-toolchain-downloads)
->          export GCC49_AARCH64_PREFIX=<path to CC>arm-gnu-toolchain-13.2.Rel1-x86_64-aarch64-none-linux-gnu/bin/aarch64-none-linux-gnu-
+>          export GCC_AARCH64_PREFIX=<path to CC>arm-gnu-toolchain-13.2.Rel1-x86_64-aarch64-none-linux-gnu/bin/aarch64-none-linux-gnu-
```

Comment:
Hi @hrw , Looks like with GCC_AARCH64_PREFIX, the build system adds new LTO compiler flags, which is causing build failures with mem consistency tests build. I would want to spend some time debugging them. To get this PR merged could you revert GCC prefix changes for mem_test in README and acsbuild.sh, and I would propose changes for mem_test when I am ready with compliation for it.

PS: We dont yet have Github actions for mem_test, would be enabling in future so this could be catched by CI.  

Thanks,
Amrathesh :)

Reply:
Hi @hrw , 
We have made changes as you suggested on top of your MR.